src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestData.scala

```diff
@@ -11,6 +13,17 @@ trait AgoraTestData {
 
   val documentation = "This is the documentation"
 
+  def getbigDocumentation: String = {
+    val readMeFile = new File("src/test/resources/", "TESTMARKDOWN.md")
```

Comment:
do we need readMeFile? I don't think you use it. (Can remove java.io.File import too I think)